modules/nf-core/plink2/het/meta.yml

```
@@ -0,0 +1,55 @@
+name: "plink2_het"
+description: Calculate Inbreeding data in plink2
+keywords:
+  - plink2
+  - inbreeding
+  - het
+tools:
+  - plink2:
+      description: |
+        Whole genome association analysis toolset, designed to perform a range
+        of basic, large-scale analyses in a computationally efficient manner
+      homepage: http://www.cog-genomics.org/plink/2.0/
+      documentation: http://www.cog-genomics.org/plink/2.0/general_usage
+      doi: "10.1186/s13742-015-0047-8"
+      licence: ["GPL v3"]
+      identifier: ""
+input:
+  - - meta:
+        type: map
+        description: |
+          Groovy Map containing sample information
+          e.g. [ id:'test' ]
+    - plink_file1:
+        type: file
+        description: PLINK binary genotype table file or PLINK 2 binary genotype table
+        pattern: "*.{bed,pgen}"
+    - plink_file2:
+        type: file
+        description: PLINK extended MAP file or PLINK 2 variant information file
+        pattern: "*.{bim,pvar}"
+    - plink_file3:
+        type: file
+        description: PLINK sample information file or PLINK 2 sample information file
+        pattern: "*.{fam,psam}"
+output:
+  - het:
+      - meta:
+         type: map
+         description: |
+           Groovy Map containing sample information
+           e.g. [ id:'test', single_end:false ]
+      - "*.het":
+         type: file
+         description: observed and expected homozygous/heterozygous genotype counts for each
+          sample and method-of-moments F coefficient estimates
+         pattern: "*.{het}"
```

```suggestion
         pattern: "*.het"
```

As it's a single file suffix

---

Fixed in commit https://github.com/nf-core/modules/pull/8081/commits/a12c4c50038b7e1193317117492539bbb2ad8acc
I've also made minor changes to the tests following review of a PR submitted by a colleague https://github.com/nf-core/modules/pull/8082